### DISCUSSION TITLE
Welcome to Laravel-Smarthr Discussions!

### DISCUSSION THREAD

**Discussion by MusheAbdulHakim** [MusheAbdulHakim/Laravel-Smarthr#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you you want to be helped with

**Comment by MACKIE199X** [comment]
Hi Mushe,

How can I activate the Module Permission I set the attendances in Employee but it won't appear in the sidebar when I logged in the employee account. 
![image](https://github.com/user-attachments/assets/aaae5109-6d37-4c46-8926-a2455d778c70)
![image](https://github.com/user-attachments/assets/455a3c15-8866-4d1a-b4ba-50f477dc84a4)

**Comment by MusheAbdulHakim** [comment]
hi @MACKIE199X 
Thanks for bringing it to my notice. 
Assigning roles to users and employees is yet to be implemented. I totally forgot about it. so I will make an update as soon as possible to this issue.

**Reply by MACKIE199X** [reply]
Thank you Mushe. Appreciate the response man. Waiting for your next update, this project is superb! 👍🫡

**Comment by MusheAbdulHakim** [comment]
Hi @MACKIE199X 
I have made the changes to resolve the issue you are having.
As of now, you can 
1. Assign user to roles
2. update user roles
3. Updates of the permissions to the various roles should reflect for the assigned users.
Feel free to reopen the discussion if you are having issues with the roles or anything related to it

**Reply by MACKIE199X** [reply]
Thank you for the update man. I'll check this out! And I'll let you know if I encounter any issues.
